db/db_impl.cc

```diff
@@ -599,7 +613,64 @@ void DBImpl::StartTimedTasks() {
             stats_dump_period_sec * 1000000));
       }
     }
+    stats_persist_period_sec = mutable_db_options_.stats_persist_period_sec;
```

Comment:
access `mutable_db_options` and (I think) `thread_persist_stats_` should hold mutex.